src/dev/flang/be/interpreter/Intrinsics.java

```diff
@@ -178,6 +179,20 @@ public static Callable call(Interpreter interpreter, Clazz innerClazz)
               return Value.EMPTY_VALUE;
             };
         });
+    put("fuzion.std.fileio.readFile", (interpreter, innerClazz)-> args ->
+        {
+          byte[] content;
+          byte[] pathBytes = (byte[])args.get(1).arrayData()._array;
+          int pos = (int)args.get(3).i32Value(); //position of the byte in file
+          Path path = Path.of(new String(pathBytes, StandardCharsets.UTF_8)); //path to the file
+          try {
+            content = Files.readAllBytes(path);
+            return new u8Value(content[pos]);
+          } catch (IOException e) {
+            System.err.println(e);
+            return new u8Value(-1);
```

Comment:
this looks like trouble. For now, just `throw new Error("some useful message "+e)` and add a comment starting `NYI` explaining that we still need proper error handling here.